src/styles/themes.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { DefaultTheme } from '@react-navigation/native';
+import { Platform } from 'react-native';
+
+import { _PRIMARY_COLOR as PRIMARY_COLOR } from '../config';
+
+/**
+ * Light theme color scheme
+ * @type {{
+ * midContrastDetail: string,
+ * errorBgColor: string,
+ * backgroundColor: string,
+ * borderColor: string,
+ * lowContrastDetail: string,
+ * textColorShadow: string,
+ * positiveBalanceColor: string,
+ * textColor: string,
+ * errorTextColor: string,
+ * tabBarBackground: string,
+ * primaryOpacity30: string,
+ * textColorShadowLight: string,
+ * primaryOpacity10: string,
+ * errorTextShadow: string,
+ * primary: string}}
+ * @property {string} backgroundColor The main background color
+ * @property {string} lowContrastDetail Low contrast with background: separator lines, containers...
+ * @property {string} midContrastDetail Medium contrast with background: placeholders, ...
+ * @property {string} borderColor Defines borders
+ * @property {string} textColor Maximum contrast with the background color, for reading
+ * @property {string} textColorShadow Washed down version of the text
+ * @property {string} textColorShadowLight More washed down version of the text
+ * @property {string} tabBarBackground Specific for tab bar selectors
+ * @property {string} positiveBalanceColor Represents a positive feedback for the user
+ * @property {string} errorBgColor For containers with error feedbacks
+ * @property {string} errorTextColor For texts with error messages
+ * @property {string} errorTextShadow Washed down version of error texts
+ * @property {string} primary Primary color, as set on the application config file
+ * @property {string} primaryOpacity10 Primary color washed down to 10% opacity
+ * @property {string} primaryOpacity30 Primary color washed down to 30% opacity
+ */
+export const COLORS = {
+  backgroundColor: '#fff',
+  lowContrastDetail: '#f7f7f7',
+  midContrastDetail: '#9e9e9e',
+  borderColor: '#eee',
+  textColor: '#000',
+  textColorShadow: 'rgba(0, 0, 0, 0.5)',
+  textColorShadowLight: 'rgba(0, 0, 0, 0.3)',
+  tabBarBackground: '#333',
+  positiveBalanceColor: '#0DA0A0',
+  errorBgColor: '#DE3535',
+  errorTextColor: '#F00',
+  errorTextShadow: `rgba(255, 0, 0, 0.7)`,
+  primary: PRIMARY_COLOR,
+  primaryOpacity10: `${PRIMARY_COLOR}1A`,
+  primaryOpacity30: `${PRIMARY_COLOR}4D`,
+};
+
+/**
+ * @type {{borderWidth: (number), headerHeight: number}}
+ * @property {number} borderWidth Defines border visibility throughout the app
+ * @property {number} headerHeight Defines the height of the screen title, for calculation purposes
+ */
+export const STYLE = {
+  borderWidth: Platform.OS === 'ios' ? 1.5 : 1, // 1 is barely visible on iOS
```

Comment:
Before the upgrade dependencies the width was 1 for both iOS and Android? Why is it different now?

Reply:
Yes, before the upgrades, the width was `1` _mostly_ on all places, some were `1.5`.

The change to `1.5` on the main style is an attempt to solve item 4 of issue 318: not being able to see lines on manual inputs. With only `1` border length on the iOS simulator the lines are barely visible, indeed.

Another solution would be to keep the border at only `1` and change the `borderColor` to have a bit more contrast with the white background.

Reply:
How it was before? The border color was in fact different?

Reply:
There's no before for iOS, so it's hard to compare. Seeing both iOS and Android screens, I clearly see the width difference, I feel it should be 1 for both. Can you share a picture of iOS with border 1, please?

Reply:
Sure! Updated the PR description with borders `1`, `1.5` and `2` for the iOS.

Reply:
Done on [c61c8e1](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/326/commits/c61c8e1d6fb377ef605aff9f59c03ed8c9798a66) :+1: